docker/ray/Dockerfile

```diff
@@ -1,11 +1,23 @@
 ARG GPU
-FROM rayproject/ray-deps:nightly"$GPU"
-ARG WHEEL_PATH
-ARG FIND_LINKS_PATH=".whl"
+ARG DOCKER_PREFIX
```

Comment:
Is there a way to keep these docker file changes a bit more minimal? 

Also @ijrsvt do you mind taking a look at this part?